.gitignore

```diff
@@ -6,3 +6,6 @@
 *.so
 .DS_Store
 .settings/**
+/target/
+.classpath
```

Comment:
Our approach has been to include our own suggested .classpath in the root directory.  That wouldn't work if we make this change

Reply:
Well, the change ignores those files from being marked for commit.

So, anyone can use his/her own `.classpath` without affecting the others.

I understand this PR doesn't have an effect, because `.classpath` is not currently in the github repository.